src/patterns/data/default/response_to_chemical_with_role_stimulus.tsv

```diff
@@ -36,3 +36,10 @@ OBA:2050352		CHEBI:35526	hypoglycemic agent	https://orcid.org/0000-0001-6757-474
 OBA:2045453	attribute of response to allergen	CHEBI:50904	allergen	https://orcid.org/0000-0001-8314-2140 
 OBA:2045454	attribute of response to thyroid hormone	CHEBI:60311	thyroid hormone	https://orcid.org/0000-0001-8314-2140 
 OBA:2050354		CHEBI:35842	antirheumatic drug	https://orcid.org/0000-0001-6757-4744
+OBA:2045551	response to non-narcotic analgesic characteristics	CHEBI:35481	non-narcotic analgesic	https://orcid.org/0000-0001-8314-2140
+OBA:2045552	response to antipyretic characteristics	CHEBI:35493	antipyretic	https://orcid.org/0000-0001-8314-2140
+OBA:2045553	prostaglandin synthetase inhibitor response characteristics	CHEBI:35544	EC 1.14.99.1 (prostaglandin-endoperoxide synthase) inhibitor	https://orcid.org/0000-0001-8314-2140
```

Comment:
This term label has a different pattern to the others.

Reply:
> This term label has a different pattern to the others.

Yes, and that is allowed.

Reply:
Ok I wanted to flag that in case you added it in error.